de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/ordercandidates/impl/OrderCandidatesRestControllerImpl.java

```diff
@@ -275,12 +275,12 @@ private OLCandCreateRequest fromJson(
 			@NonNull final MasterdataProvider masterdataProvider)
 	{
 		final String dataSourceInternalNameToUse = coalesce(
-				request.getDataSourceInternalName(),
-				DATA_SOURCE_INTERNAL_NAME);
+				request.getDataSource(),
+				"int-" + DATA_SOURCE_INTERNAL_NAME);
```

Comment:
now that i see this, it crosses my mind: please contact @pgandert1963 and coordinate with him that he prepends an `int-` to the respective taled adapters